dist/gettext.amd.js

```diff
@@ -44,10 +44,17 @@ define(function () { 'use strict';
       // sprintf equivalent, takes a string and some arguments to make a computed string
       // eg: strfmt("%1 dogs are in %2", 7, "the kitchen"); => "7 dogs are in the kitchen"
       // eg: strfmt("I like %1, bananas and %1", "apples"); => "I like apples, bananas and apples"
+      // NB: removes msg context if there is one present
       var strfmt = function (fmt) {
-        var args = arguments;
+         var args = arguments;
```

Comment:
I might be misreading but these indent formatting changes look off to me. Did you intent for this to happen?

Reply:
I dunno what I did in the past, the indentation is a total mess. (I use now spaces with 2 spaces for indent). I did not wanted to change all the indentation on the file for readability sake, but I might fix that in a near future :)